daemon/options/bpf.go

```diff
@@ -59,6 +61,7 @@ func (c *BpfConfig) ParseConfig() error {
 	if _, err = os.Stat(c.BpfFsPath); err != nil {
 		return err
 	}
+	c.VerLT513 = utils.KernelVersionLowerThan5_13()
```

Comment:
And we can remove this field as well, we can call utils.KernelVersionLowerThan5_13() where we need this value

Reply:
done